cafebabel/static/js/profile-contribution-switch.js

```diff
@@ -0,0 +1,47 @@
+const translationSwitcher = document.querySelector('.translation-switcher'),
+  articleSwitcher = document.querySelector('.article-switcher'),
```

Comment:
Use `const` on each line.

Reply:
ok, why should I use `const` on each line?